docs/events/projections/index.md

```diff
@@ -64,9 +64,9 @@ public class QuestParty
 <sup><a href='https://github.com/JasperFx/marten/blob/master/src/EventSourcingTests/Projections/QuestParty.cs#L8-L30' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_questparty' title='Start of snippet'>anchor</a></sup>
 <!-- endSnippet -->
 
-New in Marten 1.2 is the ability to use `Event<T>` metadata within your projections, assuming that you're not trying to run the aggregations inline.
+Marten provides the ability to use `IEvent<T>` metadata within your projections, assuming that you're not trying to run the aggregations inline.
 
-The syntax using the built in aggregation technique is to take in `Event<T>` as the argument to your `Apply(event)` methods,
+The syntax using the built in aggregation technique is to take in `IEvent<T>` as the argument to your `Apply(event)` methods,
 where `T` is the event type you're interested in:
 
 <!-- snippet: sample_QuestPartyWithEvents -->
```

Comment:
In the snippet no use of `IEvent<T>` is shown and I couldn't find a suitable snippet in the code-base. Do I miss something or should one be added?

Reply:
Let me check and revert.

Reply:
> The syntax using the built in aggregation technique is to take in `IEvent<T>` as the argument to your `Apply(event)` methods,
where `T` is the event type you're interested in:

Change the above as below:

The syntax using the built in aggregation technique is to take in `IEvent<T>` as the argument to your `Apply(event)` methods,
where `T` is the event type you're interested is covered in https://martendb.io/events/projections/aggregate-projections.

Reply:
I have updated it, resolving this conversation.